src/Nethermind/Nethermind.Runner/Ethereum/Steps/LoadChainspec.cs

```diff
@@ -40,12 +43,16 @@ public Task Execute()
         
         private void LoadChainSpec()
         {
+
             IInitConfig initConfig = _context.Config<IInitConfig>();
-            if (_context.Logger.IsInfo) _context.Logger.Info($"Loading chain spec from {initConfig.ChainSpecPath}");
+
+            string chainspecDir = !string.IsNullOrEmpty(initConfig.ChainSpecDirectory) ? Path.Combine(initConfig.ChainSpecDirectory, Path.GetFileName(initConfig.ChainSpecPath)) : initConfig.ChainSpecPath;
```

Comment:
there is a clear definition of meanings:

file: file only
path: both directory and file
dir: just dir

we do not want to have two separate configs for path and dir for chainspec - just one config
since it is always one file there is no need to have separate configs for dir and file